spec/requests/api/policies_spec.rb

```diff
@@ -352,4 +352,65 @@ def test_policy_profile_query(object, object_policy_profiles_url)
       test_policy_profile_query(template, template_policy_profiles_url)
     end
   end
+
+  context "Policy CUD actions" do
```

Comment:
I get that you're not testing the R in CRUD here, but CUD reads a little strange to me